wgpu-core/src/track/mod.rs

```diff
@@ -99,7 +99,7 @@ mod buffer;
 mod metadata;
 mod range;
 mod stateless;
-mod texture;
+pub(crate) mod texture;
```

Comment:
This feels wrong when there's a re-export block below.

Reply:
Made private again. Must have been left from an older revision.